include/transport/safe_queue.h

```diff
@@ -143,6 +143,13 @@ class safeQueue
   }
 
   /**
+   * @brief Check if queue is empty
+   *
+   * @returns True if empty, false if not
+   */
+  bool empty() const { return size() > 0; }
+
```

Comment:
If i am reading it correctly, it returns True if not empty ?
Suggest replacing 
```
return size() == 0;

```

Reply:
Thanks for catching that.  Updated.